Makefile

```diff
@@ -0,0 +1,41 @@
+build:
+	@echo Building plugin
+
+	rm -rf dist
+
+	cd server && go get github.com/Masterminds/glide
```

Comment:
`glide` is depreciated. See https://github.com/Masterminds/glide#golang-dep

You should migrate to go modules.